src/compiler/types.ts

```diff
@@ -8773,6 +8773,7 @@ namespace ts {
         newLine?: NewLineKind;
         omitTrailingSemicolon?: boolean;
         noEmitHelpers?: boolean;
+        indentation?: number | "\t"; /** number of spaces per indentation level or single tab per level */

```

Comment:
This JSDoc comment should precede the property, otherwise it will be emitted for the wrong one.

Reply:
have moved to the preceding line